jwala-integration-test/src/test/postman/jwala-collection.postman_collection.json

```diff
@@ -6319,33 +6447,104 @@
 							"formdata": [
 								{
 									"key": "name",
-									"value": "Postman-Test-JDK-Media-update",
+									"value": "{{JdkMediaName}}2",
```

Comment:
Don't create variable names that are appended with numbers, and definitely don't reference them in this manner where a number is being appended on the end.
Use variable names that are meaningful and provide context. Modern compilers are smart and will do performant variable name replacement so we don't have to.